src/Data/List/Membership/Setoid/Properties.agda

```diff
@@ -446,3 +447,18 @@ module _ (S : Setoid c ℓ) where
   ∈-∷=⁻ (here x≈z)   y≉v (there y∈) = there y∈
   ∈-∷=⁻ (there x∈xs) y≉v (here y≈z) = here y≈z
   ∈-∷=⁻ (there x∈xs) y≉v (there y∈) = there (∈-∷=⁻ x∈xs y≉v y∈)
+
+------------------------------------------------------------------------
+-- Any/All symmetry wrt _∈_/_∉_
+
+module _ (S : Setoid c ℓ) where
+
+  open Setoid S using (sym)
+  open Membership S
+
+  any∈sym :  ∀ {xs ys} → Any (_∈ ys) xs → Any (_∈ xs) ys
```

Comment:
There's got to better names somewhere. They are very similar in form to the existing lemma `All-swap` that was moved below. So how about `Any-swap-∈` and `All-swap-∉`? At least they would be consistent...

Reply:
Thanks for the suggestions. I'd taken the name directly from @mechvel 's original issue, without re-thinking them, so the suggestions are welcome, and these seem to fit much better with our house style!